src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -472,6 +472,23 @@ <h2 class="course-detail__title">
             {% endif %}
         {% endblock information %}
 
+        {% block category_additional_information %}
+            <!-- Showing all categories additional information of categories pages that have a page id -->
+
+            {% get_categories_pages_additional_information current_page.course as pages %}
+            {% if pages %}
+                <div class="course-detail__additional-info course-detail__block course-detail__row">
+                    {% with is_syllabus_property=True %}   
+                        {% for page in pages %}
+                            {% with reverse_id=page.reverse_id %}
+                                {% show_placeholder "additional_information" reverse_id %}
+                            {% endwith %}
+                        {% endfor %}                           
+                    {% endwith %}
+                </div>
+            {% endif %}    
+        {% endblock category_additional_information %}
```

Comment:
I'll put this block as the last one so under "License" block

Reply:
@jbpenrath The rationale that we have decided is:
Just before it's shown the `information` block, with specific course related information, so bellow is the more general information from the categories. The license information I think is always the least important.
But this is my opinion, if you think that we should put bellow the licenses, we could do it.
Just reply/confirm your decision and @Tiago-Salles would apply the change.

Reply:
@jbpenrath can you reply to my comment?
https://github.com/openfun/richie/pull/2517#discussion_r1848440926